Cargo.toml

```diff
@@ -5,9 +5,9 @@ edition = "2021"
 
 [dependencies]
 #swc_ecma_parser = "0.103.0"
-thiserror = "1.0.32"
+indexmap = "1.7.0"
+leb128 = "0.2.5"
```

Comment:
I think you can also drop these?

Reply:
We do still need `indexmap`, but you're right about `leb128`.